test/System.Net.Http.Formatting.Test/Handlers/ProgressMessageHandlerTest.cs

```diff
@@ -56,8 +56,10 @@ public async Task SendAsync_DoesNotInsertSendProgressWithoutEntityOrHandlerPrese
         }
 
         [Theory]
+#if !NET6_0_OR_GREATER // Temporarily ignore EmptyContent test issues.
```

Comment:
Issue link?

Reply:
The fix is coming in my next PR. My plan was to file the issue today and reference it from the new PR. But I can reorder things slightly&hellip;

Reply:
#386